src/textInput.test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import TextInput from './TextInput';
+
+describe('TextInput', function() {
+
+	const LABEL_TEXT = 'Super Hero',
+		VALUE = 'Batman',
+		NAME_ATTR = 'superhero',
+		MAX_LEN = '20',
+		ERROR_TEXT = 'Too wimpy.';
+
+	let textInputComponent,
+		inputEl;
+
+	beforeEach(() => {
+		const formAttrs = {
+			id: NAME_ATTR,
+			maxLength: MAX_LEN,
+			error: ERROR_TEXT,
+			required: true
+		};
+		textInputComponent = TestUtils.renderIntoDocument(<TextInput
+			name={NAME_ATTR}
+			label={LABEL_TEXT}
+			value={VALUE}
+			{...formAttrs} />);
+
+		inputEl = TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'input');
+	});
+
+	afterEach(() => {
+		textInputComponent = null;
+		inputEl = null;
+	});
+
+	it('exists', () => {
+		expect(() => TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'input')).not.toThrow();
+	});
+
+	it('should have a name attribute', () => {
+		expect(inputEl.name).toEqual(NAME_ATTR);
+	});
+
+	it('should have a value when one is specified', () => {
+		expect(inputEl.value).toEqual(VALUE);
+	});
+
+	it('should have a label when label is given', () => {
+		const labelEl = TestUtils.findRenderedDOMComponentWithTag(textInputComponent, 'label');
+		expect(labelEl).not.toBeNull();
```

Comment:
this won't be null. We should update the test to match exists setup https://github.com/meetup/meetup-web-components/pull/57/files#diff-0c6dbf0ccf27a396991373907ec031d3R38